src/pages/workspaces/List.js

```diff
@@ -170,6 +188,11 @@ const WorkspaceCard = pure(({
       href: canView ? Nav.getLink('workspace-dashboard', { namespace, name }) : undefined,
       style: listView ? styles.longCard : styles.shortCard
     }, [
+      Utils.switchCase(workspaceSubmissionStatus(workspace),
+        ['success', () => h(SubmissionIndicator, { shape: 'success-standard', color: colors.green[1] })],
+        ['failure', () => h(SubmissionIndicator, { shape: 'error-standard', color: colors.red[1] })],
+        ['running', () => h(SubmissionIndicator, { shape: 'clock', color: colors.darkBlue[1] })]
```

Comment:
We're using the clock for "submitted" and earlier statuses, and the sync icon for "running". I think we should match the icons. 

Also: do we want to also surface "submitted" workflows here too?

Reply:
Changed icon, thanks.

Regarding other statuses, we don't have access to that info at this level (without a backend change, anyway).